### DISCUSSION TITLE
TFT 35 V2 update 0.26.x

### DISCUSSION THREAD

**Discussion by BortSerg** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1737] [Q&A] [answered]
To change some parameters, I had to reflash my display again. But I ran into a problem that after reflashing in all sub-items of the menu, the "back" button returns to the main screen, and not to the previous menu item. The keystroke sound has also changed.

I bought the display a couple of months ago and at that time it was flashed with the latest firmware, I did not experience such problems.

Now I figured out the settings, downloaded the latest version from git and got a problem :(

**Comment by kisslorand** [comment]
Just tap shortly the Back button, it will bring you back to the previous menu. If you long press the Back button it  will bring you to the main menu. This is a feature added not so long ago.

**Reply by BortSerg** [reply]
this is all good, but it does not work as you say. If you go into any submenu and immediately press the "back" button, then everything works out. But it is worth staying in this menu for more than half a second, then by pressing the button we go to the main menu. It gets in the way. Let's say there is a situation when you need to calibrate the table before printing, and then go back and go to the extrusion menu for additional extrusion of plastic before printing. But now you just can't do this, but you need to go through different submenus twice. First, to calibrate the table, then to the main screen, and then to extrusion.
Maybe there is a parameter in the firmware for which affects the calculation of the time the button is pressed?

I already thought that I had a problem display, or the edge of the display where the "back" button is located. And he turned the display over and removed the film from it. Everything is the same. Even if you quickly poke the button, you will return to the main screen.

**Comment by kisslorand** [comment]
I just tested again in my available TFTs and it is working as expected, "Back" goes to main menu only if pressed for 1 second, otherwise it goes back to previous menu. No one else reported this issue you have.
I know someone with TFT  35 v2, I will ask him to test if he has or not the issue you're having.
BTW, this issue is better suited to be in the "Issues" section, it will get more attention from more people.

**Reply by BortSerg** [reply]
Sorry, I'm not quite an experienced git user and I don't quite understand where and what is better to publish, but I will keep in mind in the future. How can I contact you for video transmission? I will be filming how my TFT 35 v2 display works.

**Comment by oldman4U** [comment]
Hi. There has been a possible fix released for the reported issue today.

Please try it out and let us know.

Thank you

**Comment by oldman4U** [comment, marked as answer]
Please update your TFT using the official v27 following the Readme at the main page of this repository. Please help the community and close the discussion once you do not need it anymore. Thank you

**Reply by BortSerg** [reply]
@oldman4U, just got around to updates. Yes, in the 27th version of the firmware, this problem disappeared.
